cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/security/BasicSecurityProvider.java

```diff
@@ -20,9 +22,9 @@
   private String _userCredentialsFile;
 
   @Override
-  public void init(KafkaCruiseControlConfig config) {
+  public void init(KafkaCruiseControlConfig config) throws ServletException {
```

Comment:
It is not clear why (1) `BasicSecurityProvider`, (2) `DefaultRoleSecurityProvider`, or (3) `JwtSecurityProvider` have `throws ServletException` in their method signature.

Reply:
Fair thought, they shouldn't.